### ISSUE TITLE
Uncaught ReferenceError: require is not defined

### ISSUE THREAD

**Issue by jalilzade** [avocode/electron-windows-autoupdate#5]
Hi.
i coped your index.html to my own index.html
when trying to run electron app it says : 

_`Uncaught ReferenceError: require is not defined`_

**Comment by vojtatranta** [comment]
heh, looks like you launch index.html in classical browser, not via electron.

**Comment by vojtatranta** [comment]
you got to use `npm start` script to run Electron which is a browser wrapped in C++, which has bridge to Node.js inside.

**Comment by jalilzade** [comment]
No.i add your code to my  index.html then build my app from npm. then make pack of my electron app then run electron app.
i have 
`mainWindow.webContents.openDevTools();`
to see what happens behind.there is console error which says : 
Uncaught ReferenceError: require is not defined

**Comment by vojtatranta** [comment]
Maybe you did not allow node.js in the browser window, that's the only idea I have.

**Comment by sathiraumesh** [comment]
enable nodeIntergration when creating the window
```
app.on('ready', () => {
    console.log('Hello from Electron.');
    mainWindow = new BrowserWindow({webPreferences: {
        nodeIntegration: true
      }});
    mainWindow.webContents.loadFile(path.join(__dirname,"index.html"));
    mainWindow.webContents.openDevTools();
});
```

**Comment by vojtatranta** [comment]
yup, good point @sathiraumesh
